deployment/ccip/changeset/testhelpers/test_assertions.go

```diff
@@ -303,7 +292,7 @@ func ConfirmMultipleCommits(
 // If startBlock is nil, it will start watching from the latest block.
 func ConfirmCommitWithExpectedSeqNumRange(
 	t *testing.T,
-	src deployment.Chain,
+	srcSelector uint64,
 	dest deployment.Chain,
```

Comment:
nit: it will be easier to switch to same type of input parameter for src and dest both. If you are using selector for source, does it make sense to change the same for dest too?

Reply:
Yeah I'm working towards that, I think all of these base methods will probably end up being a interface that `deployment.Chain` and `deployment.SolChain` (or a wrapper) implement. That way we have a couple building blocks that are chain agnostic (send request, send token, transfer tokens, wait for events/commit/exec)

Wanted to get rid of the source chain first so that the logic only depends on the destination, that way we only need two versions of the code (evm, solana rather than evm->solana, solana->evm, evm->evm and solana->solana)